app/scripts/services/dimSettingsService.factory.js

```diff
@@ -83,6 +84,15 @@
       // self destruct timer for quality, hope we can remove this some day...
       savedSettings.disableQuality = new Date('9/19/2016') <= new Date();
 
+      // for now just override itemTags. eventually let users create own?
```

Comment:
I do think notes are valuable and are more appropriate in some situations than tags. However, I agree with @SunburnedGoose that tags are more powerful and custom tags will make DIM extremely flexible. Some Guardians have more nuanced naming conventions of stuff they want to keep (e.g. store, vault). I don't know how the Loadouts "actually" work, but I always imagined there was an internal tag-like system which assigned equipment to a Loadout.